lib/web/scripts/oneoff/oneoff.sh

```diff
@@ -17,20 +20,24 @@ ARCH=$({{.BinUname}} -m)
 trap 'rm -rf -- "$tempDir"' EXIT
 
 teleportTarballName() {
-    if [ ${OS} = "Darwin" ]; then
-        echo ${teleportFlavor}-${teleportVersion}-darwin-universal-bin.tar.gz
+    if [ "${OS}" = "Darwin" ]; then
+        if [ "$fips" = "true"]; then
+            echo "FIPS install is not supported on MacOS"
```

Comment:
Should we give an action for the user to workaround this issue?
Like suggesting them to run it in a Linux machine?

I'm saying this because users can use this script to call AWS APIs using their local machine (as long as AWS creds are available). However, if their cluster is using FIPS and they are using a Mac, then it won't work for them.
Telling them to run this same script in a Linux machine should help them.

Also, the term "install" might not be the most appropriate one, because this script might be used for something else other than installing teleport (configuring access in AWS or Azure).

```suggestion
            echo "FIPS version of Teleport is not compatible with MacOS. Please run this script in a Linux machine."
```